src/pages/help-centre/index.tsx

```diff
@@ -1,279 +1,37 @@
-import React, { useState, useEffect } from 'react'
-import { matchSorter } from 'match-sorter'
-import styled from 'styled-components'
-import { Helmet } from 'react-helmet'
-import Loadable from '@loadable/component'
-import { articles } from './_help-articles'
-import { SearchSuccess, SearchError } from './_search-results'
-import { eu_discards, getAllArticles } from './_utility'
-import { faq_schema } from './_faq-schema'
-import ArticleSectionComponent from './_article-section-component'
+import React from 'react'
+import QuestionsSection from './components/_questions-section'
+import SearchSection from './components/_search-section'
+import FaqSchema from './components/_faq-schema'
+import { DidntFindYourAnswerBanner, Community } from './components/_lazy-load'
+import { PLATFORMS, GENERAL, getQuestionsBySection } from './components/_utility'
```

Comment:
`PLATFROMS` and `GENERAL` are data sections i think, so IMO they should not get export from utility directory